AirportDictionary/AirportDictionary/Airport.cs

```diff
@@ -25,7 +19,7 @@ public partial class Airport
 
         [JsonProperty("elevation_ft")]
         [JsonConverter(typeof(ParseStringConverter))]
-        public long ElevationFt { get; set; }
+        public long ElevationFeet { get; set; }
```

Comment:
Best to spell out variable names entirely, like ElevationFeet.  If you abbreviate, the time you save in typing a few letters will quickly be lost in trying to remember what abbreviation you used.